features/support/fixtures.rb

```diff
@@ -61,6 +61,37 @@ def self.user_research_participants_service
     }
   end
 
+  def self.user_research_studios_service
+    return {
+      # `nil` values should be updated within the step when this fixture is used
+      id: nil,
```

Comment:
Sorry, final thing, is there a reason to the order of keys in this example?  Otherwise it would be nice to alphabetise them, maybe.